sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/Program.cs

```diff
@@ -2,6 +2,10 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Generic;
```

Comment:
since you added a bunch of using(s) here, can you ensure that none of these need to be removed in QuerySamples.cs?

Reply:
I'll take care of it in my next readme changes PR.